docs/vue/quickstart.md

```diff
@@ -48,13 +48,13 @@ So, if you’d prefer to use JavaScript instead of TypeScript, you can. After ge
 npm uninstall --save typescript @types/jest @typescript-eslint/eslint-plugin @typescript-eslint/parser @vue/cli-plugin-typescript @vue/eslint-config-typescript
 ```
 
-2. Change all `.ts` files to `.js`. In a blank Ionic Vue app, this should only be `src/router/index.ts` and `src/main.ts`.
+2. Change all `.ts` files to `.js`. In a blank Ionic Vue app, this should only be `src/router/index.ts` and `src/main.ts`. If you're using tests, also change the extension of files in the `tests` directory.
 
-3. Remove `@vue/typescript/recommended` and `@typescript-eslint/no-explicit-any: ‘off’, `from `.eslintrc.js`.
+3. In `index.html`, change the imported `<script>` file from `/src/main.ts` to `/src/main.js`.
 
-4. Remove `Array<RouteRecordRaw>` from `src/router/index.js`.
+4. Remove `@vue/typescript/recommended` and `@typescript-eslint/no-explicit-any: ‘off’, `from `.eslintrc.js`.
 
-5. Delete the `src/shims-vue.d.ts` file.
```

Comment:
Should the `shims-vue` line still be present?

Reply:
That file doesn't exist when I create a blank vue starter.

Reply:
Ah right. That was specific to the Vue CLI applications which you'll get when using Ionic CLI <= 6: https://github.com/ionic-team/starters/blob/main/vue/base/src/shims-vue.d.ts. Developers on Ionic CLI >=7 will get Vite apps which does not have that file. Maybe we can say "Delete the `src/shims-vue.d.ts` file if it exists. This is only needed when using the Vue CLI."